backend/corpora/dataset_processing/process.py

```diff
@@ -144,7 +141,7 @@ def extract_metadata(filename):
     elif raw_layer_name == "raw.X":
         raw_layer = adata.raw.X
     else:
-        raw_layer = adata.layers[raw_layer]
+        raw_layer = adata.layers["raw_layer_name"]
```

Comment:
Should this be `raw_layer_name`? instead of `"raw_layer_name"`

Reply:
yes, fixed